distribution/changelog.txt

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Feature: [#7956, #7964] Add sprite font glyphs for Hungarian and some Czech letters.
 - Fix: [#7975] Inspection flag not cleared for rides which are set to never be inspected (Original bug).
+- Improved: [#2940] Allow mouse-dragging to set patrol area (Singleplayer only).
```

Comment:
Looks like there's a conflict here.

Reply:
Yes, i will have to rebase when this pull request has been accepted.